DSL/Resql/services/add.sql

```diff
@@ -1,6 +1,2 @@
 INSERT INTO services (name, description, service_id, ruuter_type, is_common, structure)
-VALUES (:name, :description, :service_id, :ruuter_type::ruuter_request_type, :is_common, :structure::json)
-ON CONFLICT (service_id) DO UPDATE
```

Comment:
This imported, because the service_id is auto generated, it can have conflicts

Reply:
The `service_id` is not auto-generated, it's defined as `TEXT`, plus, I've removed the unique constraint on it in [pull request #282](https://github.com/buerokratt/Service-Module/pull/282/files#diff-bed6c7e892b72aaaa6c9fb1271007ca4a0b7f25fe5b6cc9d0547de79d0f84bfd), As a result, we can now insert multiple `service_id` with the same value.